### ISSUE TITLE
Quest/EIR - Add EIR Register 

### ISSUE THREAD

**Issue by f-odhiambo** [opensrp/fhircore#1046]
**Description of feature** 

1. Create an Covax based config on Quest 
2. Add an Covax practitioner with keycloack credentials 
4. Update the register to view synced covax clients 
5. Update the default client list view to be able to  have Covax based list view properties such as patient demographics (Name, Age, Gender) and vaccine status
    - Vaccinated  - Fully Immunized 
    - Recorded  - In the use-case of a multi-dosage vaccine view the last recorded vaccination 
        - Vaccine X, Date (dd-mm-YYYY)
    - Due  - Due date for the next vaccination has lapsed 

**Bottom Navigation - Change register list**
<img src="https://user-images.githubusercontent.com/4540684/153773276-3a75d6e9-934f-4c0a-a5ff-fcb08bc26d46.png" width="200" height="400" />

**Covax Client Registry**
<img src="https://user-images.githubusercontent.com/4540684/153551903-f523153c-9a73-4a69-8372-c915634e103d.png" width="200" height="400" />

**Acceptance  Criteria** 

- [ ] 1. Add the ability to sync & view Covax register  on the Quest app based on a Config 
- [ ] 2. Add the ability to add clients to the Covax register on Quest  
- [ ] 3. Add the ability to configure the Patient list view to view patient details on the list items based on the Covax profile loaded against the signed in practitioner 
- [ ] 4. Add the ability to Filter Due Immunizations . This can be accessible from the TopBar menu item

**Comment by pld** [comment]
Is there any demand for this? Unless I'm missing something I don't believe so, I'm marking this as low priority and blocked for now

**Comment by f-odhiambo** [comment]
> Is there any demand for this? Unless I'm missing something I don't believe so, I'm marking this as low priority and blocked for now

I added this to be added to Quest but functionality not it be modified/upgraded once code is merged into the Single APK the EIR module will be deleted

**Comment by pld** [comment]
k, so no demand until we have a sense of upcoming EIR work?

**Comment by dubdabasoduba** [comment]
@f-odhiambo I think we can mark this as complete based on the FHIR Configuration work completion
